landing_page/.rubocop.yml

```diff
@@ -0,0 +1,139 @@
+AllCops:
+  Include:
+    - '**/*.rb'
+    - 'archive_posts.rake'
+  Exclude:
+    - bin/**/*
+    - config/developer_configs.rb
+    - config/developer_vars.rb
+    - db/**/*
+    - node_modules/**/*
+    - vendor/**/*
+  TargetRubyVersion: 2.5
+
+# Rails:
+#   Enabled: true
```

Comment:
Why did you commented this out? What does this clause, `Rails`, mean?

Reply:
I meant to go back and delete this. It's an unrecognised cop in rubocop so that's why I commented it out

Reply:
Resolved in commit 468cf39e6692825efc60b3b265ef0896ab6f1b4a